selfdrive/ui/translations/main_zh-CHT.ts

```diff
@@ -355,35 +355,35 @@
     <message>
         <location filename="../qt/maps/map.cc" line="656"/>
         <source>km</source>
-        <translation>km</translation>
+        <translation>公里</translation>
     </message>
     <message>
         <location filename="../qt/maps/map.cc" line="659"/>
         <source>mi</source>
-        <translation>mi</translation>
+        <translation>英里</translation>
     </message>
 </context>
 <context>
     <name>MapInstructions</name>
     <message>
         <location filename="../qt/maps/map.cc" line="421"/>
         <source> km</source>
-        <translation> km</translation>
+        <translation> 公里</translation>
     </message>
     <message>
         <location filename="../qt/maps/map.cc" line="424"/>
         <source> m</source>
-        <translation> m</translation>
+        <translation> 公尺</translation>
```

Comment:
Are the Chinese translations more recognizable than the original English text for these abreviations? Weixing said km is more recognizable for simplified

Reply:
true, I'll revert them :)